### PR TITLE
Pagination official

### PR DESCRIPTION
**Pull request by lianneg19**
Added a Kaminari gem pagination

### CODE REVIEW

File: app/views/kaminari/_paginator.html.erb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<%# The container tag
+  - available local variables
+    current_page:  a page object for the currently displayed page
+    total_pages:   total number of pages
+    per_page:      number of items to fetch per page
+    remote:        data-remote
+    paginator:     the paginator that renders the pagination tags inside
+-%>
+<%= paginator.render do -%>
```
Comment (fnarciso2rrc) [inline app/views/kaminari/_paginator.html.erb:9]: How does this work?

File: Gemfile.lock [hunk 2]
```diff
@@ -126,6 +126,18 @@ GEM
       actionview (>= 5.0.0)
       activesupport (>= 5.0.0)
     json (2.7.2)
+    kaminari (1.2.2)
```
Comment (fnarciso2rrc) [inline Gemfile.lock:129]: What is kaminari?

**Review by fnarciso2rrc — APPROVED** [review]

File: app/views/kaminari/_paginator.html.erb (hunk 1 above)
Comment (lianneg19) [reply app/views/kaminari/_paginator.html.erb:9]: This is a built in feature by Kaminari gem. Basically the code creates a navigation that allows for pagination, moving between pages, such as next, previous, first, or last.

File: Gemfile.lock (hunk 2 above)
Comment (lianneg19) [reply Gemfile.lock:129]: Kaminari is a pagination gem.

**Pull request merged**